src/interface/ttk-thirdparty-api-template.yaml

```diff
@@ -970,6 +970,222 @@ paths:
           $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/501'
         503:
           $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/503'
+  /login:
+    post:
+      tags:
+        - DFSPSim
+      description: The HTTP request `POST /login` is used to login to DFSP application by providing credentials.
+      summary: DFSP user login
+      operationId: DFSPLogin
+      parameters:
+        #Headers
+        - $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/parameters/Accept.yaml'
+        - $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/parameters/Content-Length.yaml'
+        - $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/parameters/Content-Type.yaml'
+        - $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/parameters/X-Forwarded-For.yaml'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DFSPLoginRequest'
+            example:
+              userName: 'dfspa.user.name'
+              password: 'dfspa.user.password'
+              consentRequestId: '3b346cec-47b3-4def-b870-edb255aaf6c3'
+      responses:
+        200:
```

Comment:
Perhaps this should contain a list of the accounts available to be linked (all accounts that this User has with the DFSP), and some flag on each account indicating whether or not it is pre-selected based on the `POST /consents#scopes` request body.

That way, we can show the user which accounts they are linking, and they can choose to select one or more accounts.

Reply:
@lewisdaly, please take a look at my comments given here: https://github.com/mojaloop/pisp/pull/110#pullrequestreview-571113630